doc/debugger.rst

```diff
@@ -1765,25 +1764,26 @@ Eval request (0x1e)
 
 Format::
 
-    REQ <int: 0x1e> <str: expression> [<int: level>] EOM
+    REQ <int: 0x1e> <tval: level> <str: expression> EOM
```

Comment:
I thought the same thing, but wasn't sure of a good notation for a union type.  There was no precedent for such a thing before this pull.